assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -80,6 +80,141 @@ describe( 'CurrentSurvey', () => {
 		expect( container ).toMatchSnapshot();
 	} );
 
+	it( "should render a single select question when the `question_type` is 'single_select'", async () => {
```

Comment:
Just calling it out for posterity, but like I mentioned in the PRs for #3761 and #3762: these tests look great but let's split them up in a `describe` block 👍🏻

Reply:
Done